Add manual: Change the Storage Class On an Existing Volume (#278)
harvester/docs: docs/volume/edit-volume.md

If you specify a size larger than Longhorn's capacity during the expansion, the status of the volume expansion will be stuck in `Resizing`. You can cancel the failed volume expansion by clicking the `⋮` button and selecting the `Cancel Expand` option.

![cancel-failed-volume-expansion](/img/v1.2/volume/cancel-failed-volume-expansion.png)

> ## Change the Storage Class On an Existing Volume

Reviewer: ```suggestion
## Change the Storage Class of an Existing Volume
```

> ## Change the Storage Class of an Existing Volume

Reviewer 2: s\Storage Class\StorageClass
can we change this word in all places to align with the k8s' official doc? thanks.

Author: Renamed. Screenshots in the docs and the UI also need to be updated.

> The storage class on an existing image volume cannot be changed, but for other volumes, you can change the storage class by following the steps below

Reviewer: ```suggestion
The storage class of an existing image volume cannot be changed. However, you can change the storage class of other volumes by following the steps below:
```

> The storage class of an existing image volume cannot be changed. However, you can change the storage class of other volumes by following the steps below:

Reviewer 2: ```suggestion
The StorageClass of an existing volume cannot be changed. However, you can change the StorageClass while restoring a new volume from the snapshot by following the steps below:
```

> - [Take a volume snapshot](./volume-snapshots.md#create-volume-snapshots)
> - [Using other storage classes to restore the volume snapshot](./volume-snapshots.md#restore-a-new-volume-using-volume-snapshot)

Reviewer: ```suggestion
1. [Take a volume snapshot](./volume-snapshots.md#create-volume-snapshots).
2. [Use other storage classes to restore the volume snapshot](./volume-snapshots.md#restore-a-new-volume-using-volume-snapshot).
```

> 2. Use another storage class to [restore the volume snapshot](./volume-snapshots.md#restore-a-new-volume-using-volume-snapshot).

Reviewer 2: ```suggestion
2. Select StorageClass when [restoring the volume using snapshot](./volume-snapshots.md#restore-a-new-volume-using-volume-snapshot).
```
